tests/src/test/scala/org/apache/openwhisk/core/containerpool/v2/test/ActivationClientProxyTests.scala

```diff
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 
+/*
```

Comment:
Why is this test commented?

Reply:
it was an attempt to pass the travis build to merge... but looks like it does not work